main.go

```diff
@@ -60,8 +58,6 @@ func main() {
 		logger.Fatalf("ERROR: Failed to initialise OAuth2 Proxy: %v", err)
 	}
 
-	rand.Seed(time.Now().UnixNano())
```

Comment:
Are we sure there are no other usages of rand throughout the entire project? We should always seed the rand library if it's used in the project code anywhere, if it's only used in tests then this is probably ok

Have you tried `go mod why math/rand`?

Reply:
it's used only in a single test, in the project
go mod why gives me this 
```
» go mod why math/rand
# math/rand
github.com/oauth2-proxy/oauth2-proxy/v7
net/http
math/rand
```